src/Sulu/Bundle/WebsiteBundle/Controller/DefaultController.php

```diff
@@ -45,89 +44,41 @@ public function indexAction(StructureInterface $structure, $preview = false, $pa
 
     /**
      * Creates a redirect for configured webspaces.
+     *
+     * @param Request $request
+     *
+     * @return Response
+     *
+     * @deprecated since 1.2 use SuluWebsiteBundle:Redirect:redirectWebspace instead
      */
     public function redirectWebspaceAction(Request $request)
     {
-        $url = $this->resolveRedirectUrl(
-            $request->get('redirect'),
-            $request->getUri()
-        );
+        @trigger_error('SuluWebsiteBundle:Default:redirectWebspace is deprecated since version 1.2. Use the "SuluWebsiteBundle:Redirect:redirectWebspace" action instead.', E_USER_DEPRECATED);
 
-        return new RedirectResponse($url, 301);
+        return $this->forward(
```

Comment:
yes it creates a subrequest. but we cannot call the other controller directly because its not a service. Otherwise we have to use the `ControllerReolver` which is a very deep functionality of symfony which should not really be used directly (i think)